Error: Cannot find module '@natlibfi/sru-client'

Hello developer,
I am relatively new to nodejs and I am using your sru-client package. I can install it via npm but when I try to use this package in my source code I get the following error: Error: Cannot find module '@natlibfi/sru-client'
In addition to that I have got slightly different approach for importing your sru-client. My import statement is as follows: var createSruClient = require('@natlibfi/sru-client');
Could it be that there are some files missing in the current GitHub commit?
Thanks in advance
F. Fritze

--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,5 @@
-node_modules
 npm-debug.log
+node_modules
+coverage
 dist
+.nyc_output

--- a/.npmignore
+++ b/.npmignore
@@ -1,0 +1,2 @@
+coverage
+.nyc_output

--- a/src/index.js
+++ b/src/index.js
@@ -42,7 +42,7 @@ export default function ({serverUrl, version, maximumRecords, recordSchema}) {
 		searchRetrieve
 	};
 
-	async function searchRetrieve(query) {
+	async function searchRetrieve(query) { // eslint-disable-line require-await
 		return pump();
 
 		async function pump(startRecord = 1, results = []) {

